data/strings/strings.txt

```diff
@@ -9497,7 +9497,7 @@
     ru = Круглосуточно
     bg = 24 часа
     ar = ليلا ونهارا
-    cs = Den a noc
+    cs = nonstop
```

Comment:
Is this English word really used in Czech? Why not "Nepřetržitě" ?

Reply:
I've considered "nepřetržitě", but I wouldn't use it in a common speech. It would rather be "nepřetržitý provoz".
Nonstop or 24/7 is more common in Czech. We have even a wiki page: https://cs.wikipedia.org/wiki/Nonstop or https://cs.wiktionary.org/wiki/nonstop
But I made it upper-case, so now it's "Nonstop".

Reply:
Cool, thanks for explaining :)